File: src/fit/ImportFixture.java
// Copyright (C) 2003,2004 by Object Mentor, Inc. All rights reserved.
// Released under the terms of the GNU General Public License version 2 or later.
package fit;

public class ImportFixture extends Fixture
{
  public void doRow(Parse row)
  {
    String packageName = row.parts.text();
    FixtureLoader.instance().addPackageToPath(packageName);
  }
}

File: dotnet/src/fit/ImportFixture.cs
// Modified or written by Object Mentor, Inc. for inclusion with FitNesse.
// Copyright (c) 2002 Cunningham & Cunningham, Inc.
// Released under the terms of the GNU General Public License version 2 or later.
using fit;

namespace fit
{
	public class ImportFixture : Fixture
	{
		public override void DoCell(Parse cell, int columnNumber)
		{
			ObjectFactory.AddNamespace(cell.Text);
		}
	}
}

